### DISCUSSION TITLE
Bad request (400) when i try to post

### DISCUSSION THREAD

**Discussion by LeonardoAraujoLisboa** [GrapesJS/grapesjs#4410] [Q&A]
I'm doing it this way: 

storageManager: {
      type: 'remote',
      autosave: true,
      autoload: true,
      contentTypeJson: true,
      storeComponents: true, 
      storeStyles: true,     
      storeHtml: true,        
      storeCss: true,
      urlLoad: `APIURL/${idPage}`,
      urlStore: `APIURL/${idPage}`,
      fetchOptions: opts => (opts.method === 'POST' ?  { method: 'PUT' } : {}),
      onStore: data => ({ id: idPage, data }),
      onLoad: result => result.data,

The request payload when I try to edit something in canvas is this:
![image](https://user-images.githubusercontent.com/79610402/175834969-741f9de8-f707-4162-a018-9492170908f9.png)

I'm also passing the api_key and the session_token in the parameters

Artf, can you help me with this, please?

**Comment by LeonardoAraujoLisboa** [comment]
And this is the response:

![image](https://user-images.githubusercontent.com/79610402/175835518-f53e8428-62b9-4020-84aa-fa738790def4.png)

**Comment by artf** [comment]
Did you check this guide https://grapesjs.com/docs/modules/Storage.html#setup-remote-storage? It was updated with the latest release
